packages/client/hmi-client/src/components/widgets/tera-chart-settings-panel.vue

```diff
@@ -186,10 +222,25 @@ const createAnnotationDebounced = _.debounce(createAnnotation, 100);
 const cancelGenerateAnnotation = () => {
 	generateAnnotationQuery.value = '';
 };
+
+const onComparisonChange = (name, event) => {
+	const activeSettings = cloneDeep(props.activeSettings as ChartSettingComparison | null);
+	if (activeSettings?.type === ChartSettingType.VARIABLE_COMPARISON && activeSettings.variableColors) {
+		activeSettings.variableColors[name] = event.target?.value;
+		emit('comparison-selection-change', {
+			id: activeSettings.id,
+			selectedVariables: activeSettings.selectedVariables,
+			variableColors: activeSettings.variableColors
+		});
+	}
+};
```

Comment:
Nit pick. Can we relocate this method and also `const variables` under `// === Settings for comparison method ===` section (line 166 - 181) for better readability? I just put all comparison chart related things there. It would be nice to have some consistency